website/docs/reference/node-selection/syntax.md

```diff
@@ -103,9 +103,21 @@ As your selection logic gets more complex, and becomes unwieldly to type out as
 consider using a [yaml selector](/reference/node-selection/yaml-selectors). You can use a predefined definition with the `--selector` flag.
 Note that when you're using `--selector`, most other flags (namely `--select` and `--exclude`) will be ignored.
 
+### Troubleshoot with the `ls` command
+
+Constructing and debugging your selection syntax can be challenging.  To get a "preview" of what will be selected, we recommend using the [`list` command](/reference/commands/list).  This command, when combined with your selection syntax, will output a list of the nodes that meet that selection criteria.  The `dbt ls` command supports all types of selection syntax arguments, for example:
+
+```bash
```

Comment:
hey @will-sargent-dbtlabs this is a great pr and what a wealth of knowledge 🙏 - thank you for sharing! I've tweaked it so it sits above the discourse community header and also added context as to what each command does - can you verify if that's technically right? i wasn't sure about the 2nd and 4th examples